src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/dto/DeliverySessionDTO.kt

```diff
@@ -135,12 +144,40 @@ data class AppointmentFeedbackDTO(
           appointment.appointmentFeedbackSubmittedAt !== null,
           appointment.appointmentFeedbackSubmittedBy?.let { AuthUserDTO.from(it) },
         )
-      } ?: AppointmentFeedbackDTO(AttendanceFeedbackDTO(null, null, null, null, null), SessionFeedbackDTO(null, null, null, null, null), false, null)
+      }
+        ?: AppointmentFeedbackDTO(
+          AttendanceFeedbackDTO(
+            null,
```

Comment:
Hi, instead of setting lots of null in the constructors, can we set the default value to null in the object